scout/server/blueprints/cases/controllers.py

```diff
@@ -557,19 +563,17 @@ def check_outdated_gene_panel(panel_obj, latest_panel):
         missing_genes, extra_genes
     """
     # Create a list of minified gene object for the case panel {hgnc_id, gene_symbol}
-    case_panel_genes = [
```

Comment:
Is there any real reason for the full comparison of these gene dicts? I have a feeling they used to be passed back to the controller at some point..